src/appState.ts

```diff
@@ -74,6 +74,8 @@ export const getDefaultAppState = (): Omit<
     fileHandle: null,
     collaborators: new Map(),
     showStats: false,
+    currentChartType: "bar",
+    lastImportedChartTimestamp: 0,
```

Comment:
does this need to be in appState?

Reply:
It's also used in the [Stats](https://github.com/excalidraw/excalidraw/pull/2658/files#diff-4aed1253c333f6c851aaa67e1bf79897ee7c056a67e487b1a577a74761846350R89) to show the chart related info when someone inserted a chart.

It's not exported and could be removed when we will switch to chart previews before pasting (or anything else)